app/models/ThriftMe.sql

```diff
@@ -157,3 +157,11 @@ VALUES
     ('real_person', 'password1', 'user1@example.com', E'\\x89504e470d0a1a0a', true), -- Hexadecimal representation of a profile picture
     ('real_shopper', 'password2', 'user2@example.com', E'\\x89504e470d0a1a0a', true),
     ('real_person_community', 'password3', 'user3@example.com', E'\\x89504e470d0a1a0a', true);
+select * from users;
+select * from community_posts;
+select * from listings;
+select * from albums;
+delete from users;
+delete from listings;
+delete from community_posts;
+delete from albums;
```

Comment:
We'll wanna get rid of the select/delete statements in the final commit but this is fine for now